docs/ramalama.1.md

```diff
@@ -29,10 +29,25 @@ used within the VM.
 
 Default settings for flags are defined in `ramalama.conf(5)`.
 
-RamaLama supports multiple AI model registries types called transports. Supported transports:
+## SECURITY
+
+### Test and run your models more securely
+
+Because RamaLama defaults to running AI models inside of rootless containers using Podman on Docker. These containers isolate the AI models from information on the underlying host. With RamaLama containers, the AI model is mounted as a volume into the container in read/only mode. This results in the process running the model, llama.cpp or vLLM, being isolated from the host.  In addition, since `ramalama run` uses the --network=none option, the container can not reach the network and leak any information out of the system. Finally, containers are run with --rm options which means that any content written during the running of the container is wiped out when the application exits.
```

Comment:
read/only typo again

Reply:
What is the typo?

Reply:
I'm wrong, sorry.

I'd never seen read/only written with a slash before, it's just a way of writing it I'm unaware of, used to seeing:

read-only or readonly

I have seen r/o before though once or twice 😄